common/src/main/java/com/microsoft/identity/common/internal/ui/webview/challengehandlers/SwitchBrowserRequestHandler.kt

```diff
@@ -42,9 +49,13 @@ class SwitchBrowserRequestHandler(
     private val activity: Activity,
     private val context: Context,
     private val customTabsManager: CustomTabsManager,
-    private val browserSelector: IBrowserSelector
+    private val browserSelector: IBrowserSelector,
+    private val spanContext: SpanContext?
 ) : IChallengeHandler<SwitchBrowserChallenge, Unit> {
 
+    val span: Span by lazy {
+        OTelUtility.createSpanFromParent(SpanName.SwitchBrowserProtocol.name, spanContext)
```

Comment:
Is there not an existing span at this point? Seems excessive to create a new span for this

Reply:
Once we launch the intent to WebView, there is no active span in this context. You can look at other features that capture telemetry at this point and then create a new span.


Example: in AzureActiveDirectoryWebViewClient
is_sso_nonce_found_in_ests_request is not captured as the span doesn't exist.
We need to create spans like we do for ProcessNonceFromEstsRedirect, ProcessCrossCloudRedirect, Fido.